documentation/book/proc-configuring-mirror-maker.adoc

```diff
@@ -0,0 +1,129 @@
+  whitelist: "my-topic|other-topic" <10>
+  resources: <11>
+    requests:
+      cpu: "8"
+      memory: 64Gi
+    limits:
+      cpu: "12"
+      memory: 128Gi
+  logging: <12>
+    type: inline
+    loggers:
+      _logger.name_: "INFO"
+  readinessProbe: <13>
+    initialDelaySeconds: 15
+    timeoutSeconds: 5
+  livenessProbe:
+    initialDelaySeconds: 15
+    timeoutSeconds: 5
+  metrics: <14>
+    lowercaseOutputName: true
+    rules:
+      - pattern: "kafka.server<type=(.+), name=(.+)PerSec\\w*><>Count"
+        name: "kafka_server_$1_$2_total"
+      - pattern: "kafka.server<type=(.+), name=(.+)PerSec\\w*,
+        topic=(.+)><>Count"
+        name: "kafka_server_$1_$2_total"
+        labels:
+          topic: "$3"
+  jvmOptions: <15>
+    "-Xmx": "8g"
+    "-Xms": "8g"
+  image: my-org/my-image:latest <16>
+  template: <17>
+      pod:
+        affinity:
+          podAntiAffinity:
+            requiredDuringSchedulingIgnoredDuringExecution:
+              - labelSelector:
+                  matchExpressions:
+                    - key: application
+                      operator: In
+                      values:
+                        - postgresql
+                        - mongodb
+                topologyKey: "kubernetes.io/hostname"
+----
++
+<1> The number of replica nodes.
+<2> Bootstrap servers for consumer and producer.
+<3> Group ID for the consumer.
+<4> The number of consumer streams.
+<5> The offset auto-commit interval in milliseconds.
+<6> TLS encryption with key names under which TLS certificates are stored in X.509 format for consumer or producer.
+<7> Authentication for consumer or producer, using the TLS mechanism, as shown here, or a SASL-based SCRAM-SHA-512 or PLAIN mechanism.
+<8> Acknowledge (`true`) or ignore (`false`) a send failure for a message.
+<9> Kafka configuration options for consumer and producer.
+<10> Topics mirrored from source to target Kafka cluster.
+<11> Requests for reservation of supported resources, currently `cpu` and `memory`, and limits to specify the maximum resources that can be consumed.
+<12> Specified loggers and log levels added directly (`inline`) or indirectly (`external`) through a ConfigMap. A custom ConfigMap must be placed under the `log4j.properties` or `log4j2.properties` key. You can set the log level to INFO, ERROR, WARN, TRACE, DEBUG, FATAL or OFF.
+<13> Healthchecks to know when to restart a container (liveness) and when a container can accept traffic (readiness).
+<14> Prometheus metrics, which are enabled with configuration for the Prometheus JMX exporter in this example. You can enable metrics without further configuration using `metrics: {}`.
+<15> JVM configuration options to optimize performance for the Virtual Machine (VM) running Kafka Mirror Maker.
+<16> Container image configuration.
+<17> Pod scheduling to avoid sharing nodes for critical workloads. Here a pod is scheduled based with anti-affinity so the pod is not scheduled on nodes with the same hostname.
```

Comment:
You list this as pod Scheduling. But this should be listed as templates / customizing deployment and link there to make it clear there are more options than just affinity.

Reply:
Okay. I'm going to drop the pod scheduling section in the concepts too and just link to the customization section from here.